mica/report/tests/test_write_report.py

```diff
@@ -9,16 +9,21 @@
 
 from .. import report
 
+user = os.environ.get('USER') or os.environ.get('LOGNAME')
```

Comment:
I think the best way to get user information is this, which you can reliably use on any system:
```
import getpass
getpass.getuser()
```